### PR TITLE
Initial public draft — Sigmoid Rebase Function Proposal

### PR DESCRIPTION
**Pull request by statuskuo**

### CODE REVIEW

File: AIPs/aip-5.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+```
+L = lower asymptote
+U = upper asymptote
+B = growth rate
+```
+
+#### 2. "Balanced" Sigmoid
+
+Although the Sigmoid is a good start, we can improve upon it by scaling supply changes such that they “balance” one another. More specifically:
```
Comment (ahnaguib) [inline AIPs/aip-5.md:99]: The modification applied here is not a scale modifier but the shape for the contraction side is constructed differently. 
```suggestion
Although the Sigmoid is a good start, we can improve upon it by modifying the supply changes such that they “balance” one another. More specifically:
```

File: AIPs/aip-5.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+* If a demand-change-factor of `A` corresponds with supply-change-factor `B`.
+* We want to enforce that a demand-change-factor of `1/A` corresponds with a supply-change-factor of `1/B`. 
+
+This way, supply reactions to equal and opposite relative changes in demand, always even eachother out. For more context, see the alternating series example in the motivation section above.
```
Comment (ahnaguib) [inline AIPs/aip-5.md:104]: typo:
```suggestion
This way, supply reactions to equal and opposite relative changes in demand, always even each other out. For more context, see the alternating series example in the motivation section above.
```

File: AIPs/aip-5.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+### Technical Specification
+<!--The technical specification should outline the public API of the changes proposed. That is, changes to any of the interfaces Ampleforth currently exposes or the creations of new ones.-->
+
+This update creates no changes to external APIs. Clients, including exchanges, who listen to AMPL’s rebase events still receive the absolute supply change integer as before. However, note that any external application that calculates the delta independently will need to update their calculation logic.
+
+#### Initial Parameter Values
+
+These parameters were set to achieve the outcomes stated above (see motivation)—while keeping the aggregate expansion rate as similar to the current protocol as possible.
+
+Recall that the contraction curve is derived from the expansion curve through the "balancing" solution above. Thus, we need only verify that the shaping parameters selected, produce an expansion curve that generally resembles the current rebasing function. Thereafter, a balanced contraction curve will follow from the balanced_sigmoid equation as defined above.
```
Comment (ahnaguib) [inline AIPs/aip-5.md:154]: "Thereafter" implies time dependency which doesn't exist
```suggestion
Recall that the contraction curve is derived from the expansion curve through the "balancing" solution above. Thus, we need only verify that the shaping parameters selected, produce an expansion curve that generally resembles the current rebasing function. based on it a balanced contraction curve is constructed from the balanced_sigmoid equation as defined above. 
```

**Comment by nithinkrishna** [comment]
@statuskuo Squash and merge as one commit when you're done

File: AIPs/aip-5.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+```
+Y = supply change %
+x = normalized price deviation
+```
+
+<p align="center">
+<img src="../assets/aip-5/basic_sigmoid_eq_2.png" alt="drawing" width="380"/>
+</p>
+
+It has shaping parameters that determine: lower asymptote, upper asymptote, and the steepness of the curve (ie: growth rate) around its origin.
```
[the remark this answers is not in the document]
Comment (statuskuo) [reply AIPs/aip-5.md:89]: good catch, merging this in manually

File: AIPs/aip-5.md (hunk 1 above)
Comment (statuskuo) [reply AIPs/aip-5.md:99]: can we say transforming?

File: AIPs/aip-5.md (hunk 2 above)
Comment (statuskuo) [reply AIPs/aip-5.md:104]: merging in

**Pull request closed without merging**